LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Pelagicore
+Copyright (c) 2016-2017 Pelagicore
```

Comment:
@jeremiah oh, good point. Never thought of it that way. Would have saved me all those updates. :D